api/controllers/calendarEntry.controller.ts

```diff
@@ -66,7 +62,7 @@ export const seedDatabaseWithEntry = async (
 export const createCalendarEntry = async (
   req: Request,
   res: Response,
-  _next: NextFunction
+  _next: NextFunction,
 ) => {
   const entry = await CalendarEntry.create(req.body as CalendarEntry);
```

Comment:
is the result expected to be of type `CalendarEntry`? If so, can we specify that, like in line 76